Website/admin/Menus/ModuleActions/ModuleActions.js

```diff
@@ -16,7 +16,7 @@
         var isShared = opts.IsShared;
         
         if (count > 0 || supportsMove) {
-            var $form = $("body form");
+            var $form = $("body#Body form#Form");

```

Comment:
This should really just be var $form = $("#Form");  The #Form should be unique on the page and therefore the rest of that selector is slowing down the page rendering since the browser must do a bunch of extra work to process the unnecessary selectors.